Spack build error

This is seen with `$ spack install neuron+python+coreneuron`
```
==> Installing neuron-7.8.2-gct2acqwbjij6oskgfgbocvjzgbttenu
==> No binary for neuron-7.8.2-gct2acqwbjij6oskgfgbocvjzgbttenu found: installing from source
==> Using cached archive: /tmp/source_cache/_source-cache/git//neuronsimulator/nrn/7.8.2.tar.gz
==> Warning: Fetching from mirror without a checksum!
  This package is normally checked out from a version control system, but it has been archived on a spack mirror.  This means we cannot know a checksum for the tarball in advance. Be sure that your connection to this mirror is secure!
==> Applied patch /p/scratch/cjzam11/kitayama1/projects/spack/var/spack/repos/builtin/packages/neuron/patch-v782-git-cmake-avx512.patch
==> neuron: Executing phase: 'cmake'
==> Error: ProcessError: Command exited with status 1:
    'cmake' '-G' 'Unix Makefiles' '-DCMAKE_INSTALL_PREFIX:STRING=/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/neuron-7.8.2-gct2acqwbjij6oskgfgbocvjzgbttenu' '-DCMAKE_BUILD_TYPE:STRING=RelWithDebInfo' '-DCMAKE_INTERPROCEDURAL_OPTIMIZATION:BOOL=OFF' '-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON' '-DCMAKE_INSTALL_RPATH_USE_LINK_PATH:BOOL=OFF' '-DCMAKE_INSTALL_RPATH:STRING=/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/neuron-7.8.2-gct2acqwbjij6oskgfgbocvjzgbttenu/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/neuron-7.8.2-gct2acqwbjij6oskgfgbocvjzgbttenu/lib64;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/gettext-0.21-rpab6sfbr436eylkwyhtww5nvrfmxq3r/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/bzip2-1.0.8-63o3y6o4ai5nopmqx53qh5cuyyvwy7zm/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libiconv-1.16-zid6uv2jyijm57tvod4c3nmmzpfp7tsq/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libxml2-2.9.10-mgjltcdszna35khm4uabrv7vhxmmjbjo/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/xz-5.2.5-5f7lza75r4a4jjnpa7a3tykaoiqq3cxl/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/zlib-1.2.11-ysmifax4vc6oqjhhhb7j7ozvdgfyvj4u/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/ncurses-6.2-vpaqqleappl2opivpyiig26rfxuncaea/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/openmpi-4.1.1-pbq2phgdmrwhciawrt2idntanpbrgdbe/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/hwloc-2.5.0-gy7ev65uzebxyacxq2xontz4td7ndp2c/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libpciaccess-0.16-6oqycjah5m3bur6m26aazz7irg7eheln/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libevent-2.1.12-pg6ehwkrdqcfwcawb77dotji6nr42jkj/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/openssl-1.1.1k-sac47ktljuygm4dd6cd6hfpbolmpttl7/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/numactl-2.0.14-kg6l5fiy4sqqrghl46xs6om4dpnjpjmp/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/python-3.8.11-hs4bqiquwae5ybiq2sn5kfrj5626g5ob/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/expat-2.4.1-or3ejddxvjavvapslms3x6pgt5bkpgoq/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libbsd-0.11.3-56b4fqejjsfor473whgauzihdlq2z5rw/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libmd-1.0.3-7icrg2knpb7x4zme62w6kqaldlldbgu2/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/gdbm-1.19-fsegsbfou5lcibm4bngb5tqrz5a6skpd/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/readline-8.1-grsfhbtbwrnhrncp2tbp6pg6rbvzhgoh/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libffi-3.3-kzbenek56ow4c6c2g4bvxrvyvdzf2wx5/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/sqlite-3.35.5-ntuatmiprc4hyogr4d4xkl3bbxrzppgi/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/util-linux-uuid-2.36.2-4v5avv4bylx6xhxiiws3ghcxu4nqvlmw/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libffi-3.3-kzbenek56ow4c6c2g4bvxrvyvdzf2wx5/lib64' '-DCMAKE_PREFIX_PATH:STRING=/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/python-3.8.11-hs4bqiquwae5ybiq2sn5kfrj5626g5ob;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/util-linux-uuid-2.36.2-4v5avv4bylx6xhxiiws3ghcxu4nqvlmw;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/sqlite-3.35.5-ntuatmiprc4hyogr4d4xkl3bbxrzppgi;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libffi-3.3-kzbenek56ow4c6c2g4bvxrvyvdzf2wx5;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/expat-2.4.1-or3ejddxvjavvapslms3x6pgt5bkpgoq;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libbsd-0.11.3-56b4fqejjsfor473whgauzihdlq2z5rw;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libmd-1.0.3-7icrg2knpb7x4zme62w6kqaldlldbgu2;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/openmpi-4.1.1-pbq2phgdmrwhciawrt2idntanpbrgdbe;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/numactl-2.0.14-kg6l5fiy4sqqrghl46xs6om4dpnjpjmp;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libevent-2.1.12-pg6ehwkrdqcfwcawb77dotji6nr42jkj;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/hwloc-2.5.0-gy7ev65uzebxyacxq2xontz4td7ndp2c;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libpciaccess-0.16-6oqycjah5m3bur6m26aazz7irg7eheln;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/gettext-0.21-rpab6sfbr436eylkwyhtww5nvrfmxq3r;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/tar-1.34-nncr54c6utab3q6i2kakh7b5akvs7erz;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libxml2-2.9.10-mgjltcdszna35khm4uabrv7vhxmmjbjo;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/xz-5.2.5-5f7lza75r4a4jjnpa7a3tykaoiqq3cxl;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/flex-2.6.3-2yazugb3d3q5g6zqrf2ewjd3ihhlutgb;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/cmake-3.21.1-abo2opeqyvflq7gsovit4ei5wxjuwwan;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/openssl-1.1.1k-sac47ktljuygm4dd6cd6hfpbolmpttl7;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/bison-3.7.6-4ixvlv6cmgbv4hjdzm3uabkr2lnxda2f;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/zlib-1.2.11-ysmifax4vc6oqjhhhb7j7ozvdgfyvj4u;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/gdbm-1.19-fsegsbfou5lcibm4bngb5tqrz5a6skpd;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/readline-8.1-grsfhbtbwrnhrncp2tbp6pg6rbvzhgoh;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/ncurses-6.2-vpaqqleappl2opivpyiig26rfxuncaea;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/bzip2-1.0.8-63o3y6o4ai5nopmqx53qh5cuyyvwy7zm;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libiconv-1.16-zid6uv2jyijm57tvod4c3nmmzpfp7tsq' '-DNRN_ENABLE_CORENEURON:BOOL=TRUE' '-DNRN_ENABLE_INTERVIEWS:BOOL=FALSE' '-DNRN_ENABLE_MPI:BOOL=TRUE' '-DNRN_ENABLE_PYTHON:BOOL=TRUE' '-DNRN_ENABLE_RX3D:BOOL=FALSE' '-DNRN_ENABLE_CORENEURON:BOOL=TRUE' '-DNRN_ENABLE_TESTS:BOOL=FALSE' '-DNRN_ENABLE_BINARY_SPECIAL=ON' '-DPYTHON_EXECUTABLE:FILEPATH=/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/python-3.8.11-hs4bqiquwae5ybiq2sn5kfrj5626g5ob/bin/python3.8' '/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-gct2acqwbjij6oskgfgbocvjzgbttenu/spack-src'

1 error found in build log:
     28    -- Sub-project : using coreneuron from from external/coreneuron
     29    -- Found Random123: /p/software/jurecadc/stages/2020/software/Java/1
           .8_292-b10-OpenJDK
     30    -- Found CLI11: /tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-
           gct2acqwbjij6oskgfgbocvjzgbttenu/spack-src/external/coreneuron/exter
           nal/CLI11
     31    -- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE)
     32    -- Doxygen not found, building docs has been disabled
     33    /p/project/cjzam11/kitayama1/dev/clang/bin/clang-format: /usr/lib64/
           libstdc++.so.6: version `GLIBCXX_3.4.26' not found (required by /p/p
           roject/cjzam11/kitayama1/dev/clang/bin/clang-format)
  >> 34    CMake Error at external/coreneuron/CMake/packages/FindClangFormat.cm
           ake:14 (string):
     35      string sub-command REGEX, mode MATCH needs at least 5 arguments to
           tal to
     36      command.
     37    Call Stack (most recent call first):
     38      external/coreneuron/CMake/ClangFormatHelper.cmake:10 (find_package
           )
     39      external/coreneuron/CMakeLists.txt:88 (include)
     40

See build log for details:
  /tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-gct2acqwbjij6oskgfgbocvjzgbttenu/spack-build-out.txt
```

I load `clang` , but it fails:

```

==> Installing neuron-7.8.2-gct2acqwbjij6oskgfgbocvjzgbttenu
==> No binary for neuron-7.8.2-gct2acqwbjij6oskgfgbocvjzgbttenu found: installing from source
==> Applied patch /p/scratch/cjzam11/kitayama1/projects/spack/var/spack/repos/builtin/packages/neuron/patch-v782-git-cmake-avx512.patch
==> neuron: Executing phase: 'cmake'
==> Error: ProcessError: Command exited with status 1:
    'cmake' '-G' 'Unix Makefiles' '-DCMAKE_INSTALL_PREFIX:STRING=/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/neuron-7.8.2-gct2acqwbjij6oskgfgbocvjzgbttenu' '-DCMAKE_BUILD_TYPE:STRING=RelWithDebInfo' '-DCMAKE_INTERPROCEDURAL_OPTIMIZATION:BOOL=OFF' '-DCMAKE_VERBOSE_MAKEFILE:BOOL=ON' '-DCMAKE_INSTALL_RPATH_USE_LINK_PATH:BOOL=OFF' '-DCMAKE_INSTALL_RPATH:STRING=/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/neuron-7.8.2-gct2acqwbjij6oskgfgbocvjzgbttenu/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/neuron-7.8.2-gct2acqwbjij6oskgfgbocvjzgbttenu/lib64;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/gettext-0.21-rpab6sfbr436eylkwyhtww5nvrfmxq3r/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/bzip2-1.0.8-63o3y6o4ai5nopmqx53qh5cuyyvwy7zm/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libiconv-1.16-zid6uv2jyijm57tvod4c3nmmzpfp7tsq/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libxml2-2.9.10-mgjltcdszna35khm4uabrv7vhxmmjbjo/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/xz-5.2.5-5f7lza75r4a4jjnpa7a3tykaoiqq3cxl/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/zlib-1.2.11-ysmifax4vc6oqjhhhb7j7ozvdgfyvj4u/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/ncurses-6.2-vpaqqleappl2opivpyiig26rfxuncaea/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/openmpi-4.1.1-pbq2phgdmrwhciawrt2idntanpbrgdbe/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/hwloc-2.5.0-gy7ev65uzebxyacxq2xontz4td7ndp2c/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libpciaccess-0.16-6oqycjah5m3bur6m26aazz7irg7eheln/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libevent-2.1.12-pg6ehwkrdqcfwcawb77dotji6nr42jkj/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/openssl-1.1.1k-sac47ktljuygm4dd6cd6hfpbolmpttl7/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/numactl-2.0.14-kg6l5fiy4sqqrghl46xs6om4dpnjpjmp/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/python-3.8.11-hs4bqiquwae5ybiq2sn5kfrj5626g5ob/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/expat-2.4.1-or3ejddxvjavvapslms3x6pgt5bkpgoq/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libbsd-0.11.3-56b4fqejjsfor473whgauzihdlq2z5rw/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libmd-1.0.3-7icrg2knpb7x4zme62w6kqaldlldbgu2/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/gdbm-1.19-fsegsbfou5lcibm4bngb5tqrz5a6skpd/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/readline-8.1-grsfhbtbwrnhrncp2tbp6pg6rbvzhgoh/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libffi-3.3-kzbenek56ow4c6c2g4bvxrvyvdzf2wx5/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/sqlite-3.35.5-ntuatmiprc4hyogr4d4xkl3bbxrzppgi/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/util-linux-uuid-2.36.2-4v5avv4bylx6xhxiiws3ghcxu4nqvlmw/lib;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libffi-3.3-kzbenek56ow4c6c2g4bvxrvyvdzf2wx5/lib64' '-DCMAKE_PREFIX_PATH:STRING=/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/python-3.8.11-hs4bqiquwae5ybiq2sn5kfrj5626g5ob;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/util-linux-uuid-2.36.2-4v5avv4bylx6xhxiiws3ghcxu4nqvlmw;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/sqlite-3.35.5-ntuatmiprc4hyogr4d4xkl3bbxrzppgi;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libffi-3.3-kzbenek56ow4c6c2g4bvxrvyvdzf2wx5;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/expat-2.4.1-or3ejddxvjavvapslms3x6pgt5bkpgoq;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libbsd-0.11.3-56b4fqejjsfor473whgauzihdlq2z5rw;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libmd-1.0.3-7icrg2knpb7x4zme62w6kqaldlldbgu2;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/openmpi-4.1.1-pbq2phgdmrwhciawrt2idntanpbrgdbe;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/numactl-2.0.14-kg6l5fiy4sqqrghl46xs6om4dpnjpjmp;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libevent-2.1.12-pg6ehwkrdqcfwcawb77dotji6nr42jkj;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/hwloc-2.5.0-gy7ev65uzebxyacxq2xontz4td7ndp2c;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libpciaccess-0.16-6oqycjah5m3bur6m26aazz7irg7eheln;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/gettext-0.21-rpab6sfbr436eylkwyhtww5nvrfmxq3r;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/tar-1.34-nncr54c6utab3q6i2kakh7b5akvs7erz;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libxml2-2.9.10-mgjltcdszna35khm4uabrv7vhxmmjbjo;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/xz-5.2.5-5f7lza75r4a4jjnpa7a3tykaoiqq3cxl;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/flex-2.6.3-2yazugb3d3q5g6zqrf2ewjd3ihhlutgb;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/cmake-3.21.1-abo2opeqyvflq7gsovit4ei5wxjuwwan;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/openssl-1.1.1k-sac47ktljuygm4dd6cd6hfpbolmpttl7;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/bison-3.7.6-4ixvlv6cmgbv4hjdzm3uabkr2lnxda2f;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/zlib-1.2.11-ysmifax4vc6oqjhhhb7j7ozvdgfyvj4u;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/gdbm-1.19-fsegsbfou5lcibm4bngb5tqrz5a6skpd;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/readline-8.1-grsfhbtbwrnhrncp2tbp6pg6rbvzhgoh;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/ncurses-6.2-vpaqqleappl2opivpyiig26rfxuncaea;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/bzip2-1.0.8-63o3y6o4ai5nopmqx53qh5cuyyvwy7zm;/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/libiconv-1.16-zid6uv2jyijm57tvod4c3nmmzpfp7tsq' '-DNRN_ENABLE_CORENEURON:BOOL=TRUE' '-DNRN_ENABLE_INTERVIEWS:BOOL=FALSE' '-DNRN_ENABLE_MPI:BOOL=TRUE' '-DNRN_ENABLE_PYTHON:BOOL=TRUE' '-DNRN_ENABLE_RX3D:BOOL=FALSE' '-DNRN_ENABLE_CORENEURON:BOOL=TRUE' '-DNRN_ENABLE_TESTS:BOOL=FALSE' '-DNRN_ENABLE_BINARY_SPECIAL=ON' '-DPYTHON_EXECUTABLE:FILEPATH=/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-10.3.0/python-3.8.11-hs4bqiquwae5ybiq2sn5kfrj5626g5ob/bin/python3.8' '/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-gct2acqwbjij6oskgfgbocvjzgbttenu/spack-src'

1 error found in build log:
     29    -- Sub-project : using coreneuron from from external/coreneuron
     30    -- Found Random123: /p/software/jurecadc/stages/2020/software/Java/1.8_292-b10-Op
           enJDK
     31    -- Found CLI11: /tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-gct2acqwbjij6
           oskgfgbocvjzgbttenu/spack-src/external/coreneuron/external/CLI11
     32    -- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE)
     33    -- Doxygen not found, building docs has been disabled
     34    /p/project/cjzam11/kitayama1/dev/clang/bin/clang-format: /usr/lib64/libstdc++.so.
           6: version `GLIBCXX_3.4.26' not found (required by /p/project/cjzam11/kitayama1/d
           ev/clang/bin/clang-format)
  >> 35    CMake Error at external/coreneuron/CMake/packages/FindClangFormat.cmake:14 (strin
           g):
     36      string sub-command REGEX, mode MATCH needs at least 5 arguments total to
     37      command.
     38    Call Stack (most recent call first):
     39      external/coreneuron/CMake/ClangFormatHelper.cmake:10 (find_package)
     40      external/coreneuron/CMakeLists.txt:88 (include)
     41

See build log for details:
  /tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-gct2acqwbjij6oskgfgbocvjzgbttenu/spack-build-out.txt
```

```

[+] /p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-9.3.0/openmpi-4.1.1-4kd435zk7nrbhtanpzd5s4wu3qumfgbc
==> Installing py-numpy-1.21.1-oafj2gyb2ixxqngwhz3hothaqhxcg5bk
==> No binary for py-numpy-1.21.1-oafj2gyb2ixxqngwhz3hothaqhxcg5bk found: installing from source
==> Using cached archive: /tmp/source_cache/_source-cache/archive/df/dff4af63638afcc57a3dfb9e4b26d434a7a602d225b42d746ea7fe2edf1342fd.zip
==> Applied patch /p/scratch/cjzam11/kitayama1/projects/spack/var/spack/repos/builtin/packages/py-numpy/check_executables.patch
==> py-numpy: Executing phase: 'build'
==> py-numpy: Executing phase: 'install'
==> py-numpy: Successfully installed py-numpy-1.21.1-oafj2gyb2ixxqngwhz3hothaqhxcg5bk
  Fetch: 0.01s.  Build: 3m 8.57s.  Total: 3m 8.58s.
[+] /p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-9.3.0/py-numpy-1.21.1-oafj2gyb2ixxqngwhz3hothaqhxcg5bk
==> Installing neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae
==> No binary for neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae found: installing from source
==> Using cached archive: /tmp/source_cache/_source-cache/git//neuronsimulator/nrn/7.8.2.tar.gz
==> Warning: Fetching from mirror without a checksum!
  This package is normally checked out from a version control system, but it has been archived on a spack mirror.  This means we cannot know a checksum for the tarball in advance. Be sure that your connection to this mirror is secure!
==> Applied patch /p/scratch/cjzam11/kitayama1/projects/spack/var/spack/repos/builtin/packages/neuron/patch-v782-git-cmake-avx512.patch
==> neuron: Executing phase: 'cmake'
==> neuron: Executing phase: 'build'
==> Error: ProcessError: Command exited with status 2:
    'make' '-j16'

9 errors found in build log:
     1253    cd /tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktr
             qyh25mdpt6p66xugotvae/spack-build-6vz4g7x/src/nrniv && /p/scratch/
             cjzam11/kitayama1/projects/spack/lib/spack/env/gcc/gcc -DHAVE_CONF
             IG_H -Dnrniv_lib_EXPORTS -I/tmp/kitayama1/spack-stage/spack-stage-
             neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/ivoc -
             I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqy
             h25mdpt6p66xugotvae/spack-src/src/nrniv -I/tmp/kitayama1/spack-sta
             ge/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack
             -src/src/nrnoc -I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8
             .2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/oc -I/tmp/kitaya
             ma1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66x
             ugotvae/spack-build-6vz4g7x -I/tmp/kitayama1/spack-stage/spack-sta
             ge-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-build-6vz4g
             7x/src/nrncvode -I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.
             8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-build-6vz4g7x/src/nrnja
             va -I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyk
             trqyh25mdpt6p66xugotvae/spack-build-6vz4g7x/src/nrnoc -I/tmp/kitay
             ama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66
             xugotvae/spack-build-6vz4g7x/src/nrnpython -I/tmp/kitayama1/spack-
             stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/sp
             ack-build-6vz4g7x/src/oc -I/tmp/kitayama1/spack-stage/spack-stage-
             neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-build-6vz4g7x/
             src/parallel -I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2
             -6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-build-6vz4g7x/src/sundials
              -I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktr
             qyh25mdpt6p66xugotvae/spack-build-6vz4g7x/src/sundials/shared -I/t
             mp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25
             mdpt6p66xugotvae/spack-src/src -I/tmp/kitayama1/spack-stage/spack-
             stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/
             gnu -I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xy
             ktrqyh25mdpt6p66xugotvae/spack-src/src/memacs -I/tmp/kitayama1/spa
             ck-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae
             /spack-src/src/mesch -I/tmp/kitayama1/spack-stage/spack-stage-neur
             on-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/nrncvode -
             I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqy
             h25mdpt6p66xugotvae/spack-src/src/nrnmpi -I/tmp/kitayama1/spack-st
             age/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spac
             k-src/src/nrnpython -I/tmp/kitayama1/spack-stage/spack-stage-neuro
             n-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/parallel -I
             /tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh
             25mdpt6p66xugotvae/spack-src/src/sparse -I/tmp/kitayama1/spack-sta
             ge/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack
             -src/src/sparse13 -I/tmp/kitayama1/spack-stage/spack-stage-neuron-
             7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/sundials -I/t
             mp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25
             mdpt6p66xugotvae/spack-src/src/sundials/cvodes -I/tmp/kitayama1/sp
             ack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotva
             e/spack-src/src/sundials/ida -I/tmp/kitayama1/spack-stage/spack-st
             age-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/su
             ndials/shared -I/p/scratch/cjzam11/kitayama1/projects/spack/opt/sp
             ack/linux-centos8-zen2/gcc-9.3.0/python-3.8.11-bkqidzahz2gv4jlf7rj
             c7jkxlrow2npx/include/python3.8 -I/p/scratch/cjzam11/kitayama1/pro
             jects/spack/opt/spack/linux-centos8-zen2/gcc-9.3.0/openmpi-4.1.1-4
             kd435zk7nrbhtanpzd5s4wu3qumfgbc/include -I/tmp/kitayama1/spack-sta
             ge/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack
             -src/src/nrniv/nrniv_lib -I/tmp/kitayama1/spack-stage/spack-stage-
             neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/ivos -
             I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqy
             h25mdpt6p66xugotvae/spack-build-6vz4g7x/src/ivos -g  -O2 -fPIC -MD
              -MT src/nrniv/CMakeFiles/nrniv_lib.dir/__/mesch/zvecop.c.o -MF CM
             akeFiles/nrniv_lib.dir/__/mesch/zvecop.c.o.d -o CMakeFiles/nrniv_l
             ib.dir/__/mesch/zvecop.c.o -c /tmp/kitayama1/spack-stage/spack-sta
             ge-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/mes
             ch/zvecop.c
     1254    [INFO] Running make with params: ROOT=/tmp/kitayama1/spack-stage/s
             pack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-bui
             ld-6vz4g7x MODS_PATH=/tmp/kitayama1/spack-stage/spack-stage-neuron
             -7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/external/coreneu
             ron/tests/integration/ring_gap/mod
     1255    make[3]: Entering directory '/tmp/kitayama1/spack-stage/spack-stag
             e-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-build-6vz4g7
             x/bin'
     1256    make[3]: warning: -jN forced in submake: disabling jobserver mode.
     1257     -> Generating x86_64/core/mod2c/_mod_func.cpp
     1258     -> MOD2C /tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4
             g7xyktrqyh25mdpt6p66xugotvae/spack-src/external/coreneuron/tests/i
             ntegration/ring_gap/mod/halfgap.mod
  >> 1259    diff: x86_64/core/mod2c/_mod_func.cpp: No such file or directory
     1260    Replacing mod_func.c
     1261    [ 69%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrn
             oc/apcount.c.o
     1262    cd /tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktr
             qyh25mdpt6p66xugotvae/spack-build-6vz4g7x/src/nrniv && /p/scratch/
             cjzam11/kitayama1/projects/spack/lib/spack/env/gcc/gcc -DHAVE_CONF
             IG_H -Dnrniv_lib_EXPORTS -I/tmp/kitayama1/spack-stage/spack-stage-
             neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/ivoc -
             I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqy
             h25mdpt6p66xugotvae/spack-src/src/nrniv -I/tmp/kitayama1/spack-sta
             ge/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack
             -src/src/nrnoc -I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8
             .2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/oc -I/tmp/kitaya
             ma1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66x
             ugotvae/spack-build-6vz4g7x -I/tmp/kitayama1/spack-stage/spack-sta
             ge-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-build-6vz4g
             7x/src/nrncvode -I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.
             8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-build-6vz4g7x/src/nrnja
             va -I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyk
             trqyh25mdpt6p66xugotvae/spack-build-6vz4g7x/src/nrnoc -I/tmp/kitay
             ama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66
             xugotvae/spack-build-6vz4g7x/src/nrnpython -I/tmp/kitayama1/spack-
             stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/sp
             ack-build-6vz4g7x/src/oc -I/tmp/kitayama1/spack-stage/spack-stage-
             neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-build-6vz4g7x/
             src/parallel -I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2
             -6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-build-6vz4g7x/src/sundials
              -I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktr
             qyh25mdpt6p66xugotvae/spack-build-6vz4g7x/src/sundials/shared -I/t
             mp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25
             mdpt6p66xugotvae/spack-src/src -I/tmp/kitayama1/spack-stage/spack-
             stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/
             gnu -I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xy
             ktrqyh25mdpt6p66xugotvae/spack-src/src/memacs -I/tmp/kitayama1/spa
             ck-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae
             /spack-src/src/mesch -I/tmp/kitayama1/spack-stage/spack-stage-neur
             on-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/nrncvode -
             I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqy
             h25mdpt6p66xugotvae/spack-src/src/nrnmpi -I/tmp/kitayama1/spack-st
             age/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spac
             k-src/src/nrnpython -I/tmp/kitayama1/spack-stage/spack-stage-neuro
             n-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/parallel -I
             /tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh
             25mdpt6p66xugotvae/spack-src/src/sparse -I/tmp/kitayama1/spack-sta
             ge/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack
             -src/src/sparse13 -I/tmp/kitayama1/spack-stage/spack-stage-neuron-
             7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/sundials -I/t
             mp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25
             mdpt6p66xugotvae/spack-src/src/sundials/cvodes -I/tmp/kitayama1/sp
             ack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotva
             e/spack-src/src/sundials/ida -I/tmp/kitayama1/spack-stage/spack-st
             age-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/su
             ndials/shared -I/p/scratch/cjzam11/kitayama1/projects/spack/opt/sp
             ack/linux-centos8-zen2/gcc-9.3.0/python-3.8.11-bkqidzahz2gv4jlf7rj
             c7jkxlrow2npx/include/python3.8 -I/p/scratch/cjzam11/kitayama1/pro
             jects/spack/opt/spack/linux-centos8-zen2/gcc-9.3.0/openmpi-4.1.1-4
             kd435zk7nrbhtanpzd5s4wu3qumfgbc/include -I/tmp/kitayama1/spack-sta
             ge/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack
             -src/src/nrniv/nrniv_lib -I/tmp/kitayama1/spack-stage/spack-stage-
             neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/ivos -
             I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqy
             h25mdpt6p66xugotvae/spack-build-6vz4g7x/src/ivos -g  -O2 -fPIC -MD
              -MT src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/apcount.c.o -MF C
             MakeFiles/nrniv_lib.dir/__/nrnoc/apcount.c.o.d -o CMakeFiles/nrniv
             _lib.dir/__/nrnoc/apcount.c.o -c /tmp/kitayama1/spack-stage/spack-
             stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/
             nrnoc/apcount.c
     1263    [ 70%] Building C object src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrn
             oc/feature.c.o
     1264    cd /tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktr
             qyh25mdpt6p66xugotvae/spack-build-6vz4g7x/src/nrniv && /p/scratch/
             cjzam11/kitayama1/projects/spack/lib/spack/env/gcc/gcc -DHAVE_CONF
             IG_H -Dnrniv_lib_EXPORTS -I/tmp/kitayama1/spack-stage/spack-stage-
             neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/ivoc -
             I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqy
             h25mdpt6p66xugotvae/spack-src/src/nrniv -I/tmp/kitayama1/spack-sta
             ge/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack
             -src/src/nrnoc -I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8
             .2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/oc -I/tmp/kitaya
             ma1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66x
             ugotvae/spack-build-6vz4g7x -I/tmp/kitayama1/spack-stage/spack-sta
             ge-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-build-6vz4g
             7x/src/nrncvode -I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.
             8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-build-6vz4g7x/src/nrnja
             va -I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyk
             trqyh25mdpt6p66xugotvae/spack-build-6vz4g7x/src/nrnoc -I/tmp/kitay
             ama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66
             xugotvae/spack-build-6vz4g7x/src/nrnpython -I/tmp/kitayama1/spack-
             stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/sp
             ack-build-6vz4g7x/src/oc -I/tmp/kitayama1/spack-stage/spack-stage-
             neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-build-6vz4g7x/
             src/parallel -I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2
             -6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-build-6vz4g7x/src/sundials
              -I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktr
             qyh25mdpt6p66xugotvae/spack-build-6vz4g7x/src/sundials/shared -I/t
             mp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25
             mdpt6p66xugotvae/spack-src/src -I/tmp/kitayama1/spack-stage/spack-
             stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/
             gnu -I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xy
             ktrqyh25mdpt6p66xugotvae/spack-src/src/memacs -I/tmp/kitayama1/spa
             ck-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae
             /spack-src/src/mesch -I/tmp/kitayama1/spack-stage/spack-stage-neur
             on-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/nrncvode -
             I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqy
             h25mdpt6p66xugotvae/spack-src/src/nrnmpi -I/tmp/kitayama1/spack-st
             age/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spac
             k-src/src/nrnpython -I/tmp/kitayama1/spack-stage/spack-stage-neuro
             n-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/parallel -I
             /tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh
             25mdpt6p66xugotvae/spack-src/src/sparse -I/tmp/kitayama1/spack-sta
             ge/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack
             -src/src/sparse13 -I/tmp/kitayama1/spack-stage/spack-stage-neuron-
             7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/sundials -I/t
             mp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25
             mdpt6p66xugotvae/spack-src/src/sundials/cvodes -I/tmp/kitayama1/sp
             ack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotva
             e/spack-src/src/sundials/ida -I/tmp/kitayama1/spack-stage/spack-st
             age-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/su
             ndials/shared -I/p/scratch/cjzam11/kitayama1/projects/spack/opt/sp
             ack/linux-centos8-zen2/gcc-9.3.0/python-3.8.11-bkqidzahz2gv4jlf7rj
             c7jkxlrow2npx/include/python3.8 -I/p/scratch/cjzam11/kitayama1/pro
             jects/spack/opt/spack/linux-centos8-zen2/gcc-9.3.0/openmpi-4.1.1-4
             kd435zk7nrbhtanpzd5s4wu3qumfgbc/include -I/tmp/kitayama1/spack-sta
             ge/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack
             -src/src/nrniv/nrniv_lib -I/tmp/kitayama1/spack-stage/spack-stage-
             neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/ivos -
             I/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqy
             h25mdpt6p66xugotvae/spack-build-6vz4g7x/src/ivos -g  -O2 -fPIC -MD
              -MT src/nrniv/CMakeFiles/nrniv_lib.dir/__/nrnoc/feature.c.o -MF C
             MakeFiles/nrniv_lib.dir/__/nrnoc/feature.c.o.d -o CMakeFiles/nrniv
             _lib.dir/__/nrnoc/feature.c.o -c /tmp/kitayama1/spack-stage/spack-
             stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/
             nrnoc/feature.c
     1265    Translating  into x86_64/core/mod2c//halfgap.cpp

     ...

     1659    Building python module with: /p/scratch/cjzam11/kitayama1/projects
             /spack/opt/spack/linux-centos8-zen2/gcc-9.3.0/python-3.8.11-bkqidz
             ahz2gv4jlf7rjc7jkxlrow2npx/bin/python3.8
     1660    cd /tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktr
             qyh25mdpt6p66xugotvae/spack-build-6vz4g7x/src/nrnpython && /p/scra
             tch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/
             gcc-9.3.0/cmake-3.21.1-tsxbmhroyx77nyzw3gduutpxg6iukxe3/bin/cmake
             -E copy_if_different /tmp/kitayama1/spack-stage/spack-stage-neuron
             -7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-src/src/nrnpython/in
             ithoc.cpp /tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4
             g7xyktrqyh25mdpt6p66xugotvae/spack-build-6vz4g7x/src/nrnpython/ini
             thoc.cpp
     1661    cd /tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktr
             qyh25mdpt6p66xugotvae/spack-build-6vz4g7x/src/nrnpython && /p/scra
             tch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/
             gcc-9.3.0/python-3.8.11-bkqidzahz2gv4jlf7rjc7jkxlrow2npx/bin/pytho
             n3.8 setup.py --quiet build --build-lib=/tmp/kitayama1/spack-stage
             /spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-b
             uild-6vz4g7x/lib/python
     1662    [100%] Linking CXX executable ../../bin/nrniv
     1663    cd /tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktr
             qyh25mdpt6p66xugotvae/spack-build-6vz4g7x/src/nrniv && /p/scratch/
             cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-
             9.3.0/cmake-3.21.1-tsxbmhroyx77nyzw3gduutpxg6iukxe3/bin/cmake -E c
             make_link_script CMakeFiles/nrniv.dir/link.txt --verbose=1
     1664    /p/scratch/cjzam11/kitayama1/projects/spack/lib/spack/env/gcc/g++
             -g  -O2 CMakeFiles/nrniv.dir/__/ivoc/nrnmain.cpp.o CMakeFiles/nrni
             v.dir/__/oc/ockludge.c.o CMakeFiles/nrniv.dir/__/oc/modlreg.c.o -o
              ../../bin/nrniv  -Wl,-rpath,/tmp/kitayama1/spack-stage/spack-stag
             e-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-build-6vz4g7
             x/lib:/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-
             centos8-zen2/gcc-9.3.0/readline-8.1-uxmzaf75tn6vtss35jw7iiwaisamdt
             tk/lib:/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux
             -centos8-zen2/gcc-9.3.0/ncurses-6.2-vtu6subobiwbi3iz7lsp2n5dtksisa
             7c/lib:/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux
             -centos8-zen2/gcc-9.3.0/python-3.8.11-bkqidzahz2gv4jlf7rjc7jkxlrow
             2npx/lib:/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/lin
             ux-centos8-zen2/gcc-9.3.0/openmpi-4.1.1-4kd435zk7nrbhtanpzd5s4wu3q
             umfgbc/lib:::::::::::::::::::::::: ../../lib/libnrniv.so -pthread
             -ldl /p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-c
             entos8-zen2/gcc-9.3.0/readline-8.1-uxmzaf75tn6vtss35jw7iiwaisamdtt
             k/lib/libreadline.so /p/scratch/cjzam11/kitayama1/projects/spack/o
             pt/spack/linux-centos8-zen2/gcc-9.3.0/ncurses-6.2-vtu6subobiwbi3iz
             7lsp2n5dtksisa7c/lib/libcurses.so /p/scratch/cjzam11/kitayama1/pro
             jects/spack/opt/spack/linux-centos8-zen2/gcc-9.3.0/ncurses-6.2-vtu
             6subobiwbi3iz7lsp2n5dtksisa7c/lib/libform.so /p/scratch/cjzam11/ki
             tayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-9.3.0/pyth
             on-3.8.11-bkqidzahz2gv4jlf7rjc7jkxlrow2npx/lib/libpython3.8.so /p/
             scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-z
             en2/gcc-9.3.0/openmpi-4.1.1-4kd435zk7nrbhtanpzd5s4wu3qumfgbc/lib/l
             ibmpi.so
  >> 1665    ../../lib/libnrniv.so: error: undefined reference to 'tgoto'
  >> 1666    ../../lib/libnrniv.so: error: undefined reference to 'tgetent'
  >> 1667    ../../lib/libnrniv.so: error: undefined reference to 'tputs'
  >> 1668    ../../lib/libnrniv.so: error: undefined reference to 'tgetnum'
  >> 1669    ../../lib/libnrniv.so: error: undefined reference to 'tgetstr'
  >> 1670    collect2: error: ld returned 1 exit status
  >> 1671    make[2]: *** [src/nrniv/CMakeFiles/nrniv.dir/build.make:138: bin/n
             rniv] Error 1
     1672    make[2]: Leaving directory '/tmp/kitayama1/spack-stage/spack-stage
             -neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-build-6vz4g7x
             '
  >> 1673    make[1]: *** [CMakeFiles/Makefile2:1223: src/nrniv/CMakeFiles/nrni
             v.dir/all] Error 2
     1674    make[1]: *** Waiting for unfinished jobs....
     1675    make[2]: Leaving directory '/tmp/kitayama1/spack-stage/spack-stage
             -neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-build-6vz4g7x
             '
     1676    [100%] Built target hoc_module
     1677    make[1]: Leaving directory '/tmp/kitayama1/spack-stage/spack-stage
             -neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-build-6vz4g7x
             '
     1678    make: *** [Makefile:159: all] Error 2

See build log for details:
  /tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-build-out.txt
```

> ../../lib/libnrniv.so: error: undefined reference to 'tgoto'

Looks like now `ncurses` lib is not found, which is strange because it is available via spack (`/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-9.3.0/ncurses-6.2-vtu6subobiwbi3iz7lsp2n5dtksisa7c/lib`). Could you upload `/tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-6vz4g7xyktrqyh25mdpt6p66xugotvae/spack-build-out.txt` here?

Then it is worth retrying with a `spack clean -a`.

Nope.

```

[+] /p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-9.3.0/py-numpy-1.21.1-oafj2gyb2ixxqngwhz3hothaqhxcg5bk
==> Installing neuron-7.8.2-r4fuskj5rkri6vg66p2qapyjgnhf4xuj
==> No binary for neuron-7.8.2-r4fuskj5rkri6vg66p2qapyjgnhf4xuj found: installing from source
==> Using cached archive: /tmp/source_cache/_source-cache/git//neuronsimulator/nrn/7.8.2.tar.gz
==> Warning: Fetching from mirror without a checksum!
  This package is normally checked out from a version control system, but it has been archived on a spack mirror.  This means we cannot know a checksum for the tarball in advance. Be sure that your connection to this mirror is secure!
==> Applied patch /p/scratch/cjzam11/kitayama1/projects/spack/var/spack/repos/builtin/packages/neuron/patch-v782-git-cmake-avx512.patch
==> neuron: Executing phase: 'cmake'
==> neuron: Executing phase: 'build'
==> Error: ProcessError: Command exited with status 2:
    'make' '-j16'

8 errors found in build log:
     1211    Building python module with: /p/scratch/cjzam11/kitayama1/projects
             /spack/opt/spack/linux-centos8-zen2/gcc-9.3.0/python-3.8.11-bkqidz
             ahz2gv4jlf7rjc7jkxlrow2npx/bin/python3.8
     1212    cd /tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-r4fuskj5rkr
             i6vg66p2qapyjgnhf4xuj/spack-build-r4fuskj/src/nrnpython && /p/scra
             tch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/
             gcc-9.3.0/cmake-3.21.1-tsxbmhroyx77nyzw3gduutpxg6iukxe3/bin/cmake
             -E copy_if_different /tmp/kitayama1/spack-stage/spack-stage-neuron
             -7.8.2-r4fuskj5rkri6vg66p2qapyjgnhf4xuj/spack-src/src/nrnpython/in
             ithoc.cpp /tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-r4fu
             skj5rkri6vg66p2qapyjgnhf4xuj/spack-build-r4fuskj/src/nrnpython/ini
             thoc.cpp
     1213    cd /tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-r4fuskj5rkr
             i6vg66p2qapyjgnhf4xuj/spack-build-r4fuskj/src/nrnpython && /p/scra
             tch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/
             gcc-9.3.0/python-3.8.11-bkqidzahz2gv4jlf7rjc7jkxlrow2npx/bin/pytho
             n3.8 setup.py --quiet build --build-lib=/tmp/kitayama1/spack-stage
             /spack-stage-neuron-7.8.2-r4fuskj5rkri6vg66p2qapyjgnhf4xuj/spack-b
             uild-r4fuskj/lib/python
     1214    [100%] Linking CXX executable ../../bin/nrniv
     1215    cd /tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-r4fuskj5rkr
             i6vg66p2qapyjgnhf4xuj/spack-build-r4fuskj/src/nrniv && /p/scratch/
             cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-
             9.3.0/cmake-3.21.1-tsxbmhroyx77nyzw3gduutpxg6iukxe3/bin/cmake -E c
             make_link_script CMakeFiles/nrniv.dir/link.txt --verbose=1
     1216    /p/scratch/cjzam11/kitayama1/projects/spack/lib/spack/env/gcc/g++
             -g  -O2 CMakeFiles/nrniv.dir/__/ivoc/nrnmain.cpp.o CMakeFiles/nrni
             v.dir/__/oc/ockludge.c.o CMakeFiles/nrniv.dir/__/oc/modlreg.c.o -o
              ../../bin/nrniv  -Wl,-rpath,/tmp/kitayama1/spack-stage/spack-stag
             e-neuron-7.8.2-r4fuskj5rkri6vg66p2qapyjgnhf4xuj/spack-build-r4fusk
             j/lib:/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-
             centos8-zen2/gcc-9.3.0/readline-8.1-uxmzaf75tn6vtss35jw7iiwaisamdt
             tk/lib:/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux
             -centos8-zen2/gcc-9.3.0/ncurses-6.2-vtu6subobiwbi3iz7lsp2n5dtksisa
             7c/lib:/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux
             -centos8-zen2/gcc-9.3.0/python-3.8.11-bkqidzahz2gv4jlf7rjc7jkxlrow
             2npx/lib:/p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/lin
             ux-centos8-zen2/gcc-9.3.0/openmpi-4.1.1-4kd435zk7nrbhtanpzd5s4wu3q
             umfgbc/lib:::::::::::::::::::::::: ../../lib/libnrniv.so -pthread
             -ldl /p/scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-c
             entos8-zen2/gcc-9.3.0/readline-8.1-uxmzaf75tn6vtss35jw7iiwaisamdtt
             k/lib/libreadline.so /p/scratch/cjzam11/kitayama1/projects/spack/o
             pt/spack/linux-centos8-zen2/gcc-9.3.0/ncurses-6.2-vtu6subobiwbi3iz
             7lsp2n5dtksisa7c/lib/libcurses.so /p/scratch/cjzam11/kitayama1/pro
             jects/spack/opt/spack/linux-centos8-zen2/gcc-9.3.0/ncurses-6.2-vtu
             6subobiwbi3iz7lsp2n5dtksisa7c/lib/libform.so /p/scratch/cjzam11/ki
             tayama1/projects/spack/opt/spack/linux-centos8-zen2/gcc-9.3.0/pyth
             on-3.8.11-bkqidzahz2gv4jlf7rjc7jkxlrow2npx/lib/libpython3.8.so /p/
             scratch/cjzam11/kitayama1/projects/spack/opt/spack/linux-centos8-z
             en2/gcc-9.3.0/openmpi-4.1.1-4kd435zk7nrbhtanpzd5s4wu3qumfgbc/lib/l
             ibmpi.so
  >> 1217    ../../lib/libnrniv.so: error: undefined reference to 'tgoto'
  >> 1218    ../../lib/libnrniv.so: error: undefined reference to 'tgetent'
  >> 1219    ../../lib/libnrniv.so: error: undefined reference to 'tputs'
  >> 1220    ../../lib/libnrniv.so: error: undefined reference to 'tgetnum'
  >> 1221    ../../lib/libnrniv.so: error: undefined reference to 'tgetstr'
  >> 1222    collect2: error: ld returned 1 exit status
  >> 1223    make[2]: *** [src/nrniv/CMakeFiles/nrniv.dir/build.make:138: bin/n
             rniv] Error 1
     1224    make[2]: Leaving directory '/tmp/kitayama1/spack-stage/spack-stage
             -neuron-7.8.2-r4fuskj5rkri6vg66p2qapyjgnhf4xuj/spack-build-r4fuskj
             '
  >> 1225    make[1]: *** [CMakeFiles/Makefile2:306: src/nrniv/CMakeFiles/nrniv
             .dir/all] Error 2
     1226    make[1]: *** Waiting for unfinished jobs....
     1227    make[2]: Leaving directory '/tmp/kitayama1/spack-stage/spack-stage
             -neuron-7.8.2-r4fuskj5rkri6vg66p2qapyjgnhf4xuj/spack-build-r4fuskj
             '
     1228    [100%] Built target hoc_module
     1229    make[1]: Leaving directory '/tmp/kitayama1/spack-stage/spack-stage
             -neuron-7.8.2-r4fuskj5rkri6vg66p2qapyjgnhf4xuj/spack-build-r4fuskj
             '
     1230    make: *** [Makefile:139: all] Error 2

See build log for details:
  /tmp/kitayama1/spack-stage/spack-stage-neuron-7.8.2-r4fuskj5rkri6vg66p2qapyjgnhf4xuj/spack-build-out.txt
```

--- a/var/spack/repos/builtin/packages/neuron/package.py
+++ b/var/spack/repos/builtin/packages/neuron/package.py
@@ -21,6 +21,7 @@ class Neuron(CMakePackage):
     maintainers = ['pramodk', 'nrnhines', 'iomaganaris', 'alexsavulescu']
 
     version('develop', branch='master', submodules='True')
+    version("8.0.0", tag="8.0.0", submodules='True')
     version("7.8.2", tag="7.8.2", submodules='True')
     version("7.8.1", tag="7.8.1", submodules='True')
 
@@ -43,6 +44,7 @@ class Neuron(CMakePackage):
     depends_on("ncurses")
     depends_on("python@2.7:", when="+python")
     depends_on("py-pytest",   when="+python+tests")
+    depends_on("py-mpi4py",   when="+mpi+python+tests")
     depends_on("readline")
     depends_on("caliper",     when="+caliper")
     depends_on("py-numpy",    type='run')

